### DISCUSSION TITLE
Rapid response resources

### DISCUSSION THREAD

**Discussion by fight-against-hate** [fight-against-hate/fight-against-hate#2] [Q&A]
Who has temporary housing support for both individual and families?

Any programs for grants?  How do you donate to them?

**Comment by ghost** [comment]
PA - [Eastern PA Trans Equity Project](https://www.patransequity.org/) has [rapid response crisis grants](https://www.patransequity.org/transgender-rapid-response-crisis-grants/)

CA - [Office of Transgender Initiatives](https://sf.gov/departments/city-administrator/office-transgender-initiatives) has [housing resources](https://sf.gov/information/transgender-housing)
